en_US/data-integration/data-bridge-cassa.md

```diff
@@ -27,8 +27,9 @@ EMQX Enterprise Edition features. EMQX Enterprise Edition provides comprehensive
 
 ## Feature List
 
-- [Connection pool](./data-bridges.md)
-- [SQL preprocessing](./data-bridges.md)
+- [Connection pool](./data-bridges.md#connection-pool)
+- [Async mode](./data-bridges.md#async-mode)
+- [SQL preprocessing](./data-bridges.md#prepared-statement)
```

Comment:
It is a proper noun. Suggestion rename all **SQL preprocessing** to **SQL prepared statement**